code/modules/mob/living/carbon/carbon.dm

```diff
@@ -1184,7 +1184,8 @@ so that different stomachs can handle things in different ways VB*/
 		return
 
 	var/obj/item/organ/internal/eyes/E = get_organ_slot("eyes")
-	E.flash_protect = initial(E.flash_protect)
+	if(E)
+		E.flash_protect = initial(E.flash_protect)
```

Comment:
This means you can get the gene. Remove your eyes, mutadone yourself, replace the eyes and now you have free flash proof eyes. Could perhaps be made into a trait?

Reply:
The initial state is just the default `flash_protect` state of the eye organ, so this should not be a problem.

Reply:
oh, i understand now, hmm

Reply:
I think replacing eyes calls update_sight() so this won't be an issue, but I will check later

Reply:
Another issue with this is it is a dud gene on slimes and skeletons. Maybe not a massive issue, but it is a consideration.

Reply:
This has been fixed, it now checks for the trait during the `flash_eyes` proc. So it's eye-independent.